jpo-ode-plugins/src/main/java/us/dot/its/jpo/ode/plugin/j2735/J2735GeometricProjection.java

```diff
@@ -0,0 +1,45 @@
+package us.dot.its.jpo.ode.plugin.j2735;
+
+import us.dot.its.jpo.ode.plugin.asn1.Asn1Object;
+import us.dot.its.jpo.ode.plugin.j2735.timstorage.Circle;
+
+public class J2735GeometricProjection extends Asn1Object {
+    private static final long serialVersionUID = 1L;
+
+    private J2735HeadingSlice direction;
```

Comment:
Should be a J2735BitString here, because HeadingSlice is a BitString.  Need to be able to represent more than one value set at the same time.